src/components/Leaves/Leaves.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class="leaves">
+    <Leave v-for="team in teamNames" v-bind:key="teamNames.indexOf(team)" :team="team"></Leave>
+  </div>
+</template>
+
+<script>
+import Leave from './Leave.vue';
+
+export default {
+  name: 'Leaves',
+  components: {
+    Leave,
+  },
+  computed: {
+    teamNames() {
+      return this.teams.map(team => {
```

Comment:
This may be better expressed as a switch.

Reply:
I like this if form for this specific case, looks cleaner for me and it's more _compact_